tests/snapshots/test_app/test_chat_vision_user/auth_client0/result.json

```diff
@@ -8,8 +8,8 @@
                 "westbrae_jun28.pdf#page=2"
             ],
             "images": [
-                "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z/C/HgAGgwJ/lK3Q6wAAAABJRU5ErkJggg==",
-                "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z/C/HgAGgwJ/lK3Q6wAAAABJRU5ErkJggg=="
+                "data:image/png;base64,dGVzdCBjb250ZW50",
```

Comment:
Hm do you know why these changed?

Reply:
no, good question, we'll have to look more deeply